Example/ExampleViewController.swift

```diff
@@ -172,6 +172,9 @@ class ExampleViewController: UIViewController {
         config.library.allowedMultiSelectionAspectRatioOverrides = overrides
         config.library.allowPhotoAndVideoSelection = true
 
+        /* Set a max height for the asset preview */
+        // config.library.assetPreviewMaxHeight = 280.0
```

Comment:
Is this stuff all commented out just to show examples of different configuration options?

Reply:
Yes, exactly. I didn't want to have this new thing on as default but still have it easily available for testing. That seems to have been the style for the other options as well.